install/updateVersion.sh

```diff
@@ -37,6 +37,7 @@ for target in istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-name
     gen_file $target
 done
 
+make sidecar-injector.yaml
```

Comment:
Why not add `sidecar-injector.yaml` to the list of targets on line 36?

Reply:
Since this yaml is not generated before, copy -f will fail in the for loop.

Reply:
Let's just remove this file.